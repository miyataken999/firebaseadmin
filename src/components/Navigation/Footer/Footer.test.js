import React from 'react';
import { render } from '@testing-library/react';
import Footer from '.';

describe('<Footer /> rendering', () => {
  it('should render without crashing', () => {
    const component = render(<Footer />);

    expect(component.asFragment()).toMatchSnapshot();
  });
});
