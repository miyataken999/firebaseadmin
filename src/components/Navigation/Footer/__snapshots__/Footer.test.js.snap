// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> rendering should render without crashing 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="container-fluid"
    >
      <div
        class="level level"
      >
        <div
          class="level-left"
        >
          <div
            class="level-item"
          >
            © 2020 
            <span>
                CreateThrive
            </span>
          </div>
        </div>
        <div
          class="level-right levelRight"
        >
          <div
            class="level-item"
          >
            <div
              class="logo"
            >
              <a
                href="https://createthrive.com"
              >
                <img
                  alt="CreateThrive.com"
                  src="https://createthrive.com/assets/images/Logo-CT.svg"
                />
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
